src/components/PostInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { UserInfo } from './UserInfo';
+import { CommentList } from './CommentList';
+
+import { Post } from '../types/Post';
+
+type Props = Omit<Post, 'id'>;
+
+export const PostInfo: React.FC<Props> = ({ ...post }) => (
```

Comment:
if you use spread operator for passing props then destructure it in component arguments, not gather again with rest operator
```suggestion
export const PostInfo: React.FC<Props> = ({ title, body, user, comments }) => (
```